Purchases/Purchasing/StoreKitAbstractions/SK2StoreProductDiscount.swift

```diff
@@ -34,11 +35,11 @@ internal struct SK2StoreProductDiscount: StoreProductDiscountType {
     let underlyingSK2Discount: SK2ProductDiscount
 
     let offerIdentifier: String?
+    let currencyCode: String?
     let price: Decimal
     let paymentMode: StoreProductDiscount.PaymentMode
     let subscriptionPeriod: SubscriptionPeriod
     let type: StoreProductDiscount.DiscountType
-
```

Comment:
I don't love this either :P but our style guide says you need a line break here.